test/puppetlabs/puppetdb/testutils.clj

```diff
@@ -33,7 +33,11 @@
     `(do ~@body)
     (let [case-versions (remove keyword? (take-nth 2 bindings))]
       `(doseq ~bindings
-         (testing (str "Testing case " '~case-versions)
+         (testing (str "Testing case: " (clojure.string/join
+                                          ", "
+                                          (map #(str %1 ": " %2)
+                                               '~case-versions
+                                               (list ~@case-versions))))
```

Comment:
:+1: on the chainable prose
but I'm not sure whether the let binding style is achievable, consider using destructuring:
```clojure
(deftest some-test
  (testing "check something"
    (dotestseq [{:keys [x y]} [{:x 10 :y 20}]
                z [:a :b]]
               (is (= 13 [x y z])))))
```
then you get something like this:
```
FAIL in (some-test) (testutils.clj:45)
check something with [[{:keys [a b]} {:keys [10 20]}] [[c d] [3 4]]]
expected: (= 13 [a b c d])
  actual: (not (= 13 [10 20 3 4]))
```
which doesn't really look like a let binding